docs/data_management/lts/lts_faq.md

```diff
@@ -0,0 +1,120 @@
+# LTS FAQ
+
+## Can I Share My Account Access Keys With Other People?
+
+You should never share access keys with anyone. These should be treated similarly to your BlazerID and password. Sharing keys creates a point of vulnerability and if they fall into a nefarious actor's hands, all buckets that account owns and the data in them can be deleted.
+
+In some cases, you may not be actively managing data in a bucket even though you own the account which owns a shared bucket. Instead of sharing keys with a data steward, instead that steward should be given admin-esque permissions on the required bucket via a policy file.
```

Comment:
Let's add a reference to contacting support, this can currently only be done by Clyde (and maybe John and Mike).

Reply:
Cancel this, I misunderstood.